channeldb/invoices.go

```diff
@@ -1876,60 +1881,48 @@ func settleHtlcsAmp(invoice *invpkg.Invoice,
 
 // updateInvoice fetches the invoice, obtains the update descriptor from the
 // callback and applies the updates in a single db transaction.
-func (d *DB) updateInvoice(hash *lntypes.Hash, refSetID *invpkg.SetID, invoices,
-	settleIndex, setIDIndex kvdb.RwBucket, invoiceNum []byte,
-	callback invpkg.InvoiceUpdateCallback) (*invpkg.Invoice, error) {
-
-	// If the set ID is non-nil, then we'll use that to filter out the
-	// HTLCs for AMP invoice so we don't need to read them all out to
-	// satisfy the invoice callback below. If it's nil, then we pass in the
-	// zero set ID which means no HTLCs will be read out.
-	var invSetID invpkg.SetID
```

Comment:
Looks like this logic with the set ID isn't replicated in the call above to `fetchInvoice`.